tasks/Task.cpp

```diff
@@ -20,11 +20,11 @@ Task::~Task()
 static Eigen::Quaterniond convertToQuaterniond(PingStatus const& data)
 {
     Eigen::Quaterniond attitude = Eigen::Quaterniond(
-        Eigen::AngleAxisd(data.response.acoustic_fix.attitude_roll / 10. / 180.0 * M_PI,
+        Eigen::AngleAxisd(data.response.acoustic_fix.attitude_yaw / 10. / 180.0 * M_PI,
             Eigen::Vector3d::UnitX()) *
         Eigen::AngleAxisd(data.response.acoustic_fix.attitude_pitch / 10. / 180.0 * M_PI,
             Eigen::Vector3d::UnitY()) *
-        Eigen::AngleAxisd(data.response.acoustic_fix.attitude_yaw / 10. / 180.0 * M_PI,
+        Eigen::AngleAxisd(data.response.acoustic_fix.attitude_roll / 10. / 180.0 * M_PI,
             Eigen::Vector3d::UnitZ()));
     return attitude;
```

Comment:
This is wrong, yaw should be on Z axis and roll on X (you did convertToRBS right, just check it out) @kapeps  mentioned that you are not going to use this anymore, but I thought you should know this error was here anyway.